DXRModules/DeusEx/Classes/DXRStartMap.uc

```diff
@@ -265,6 +274,9 @@ static function int GetStartMapMission(int start_map_val)
         case 99:
             mission=10; //Mission 9 graveyard, basically mission 10
             break;
+        case 109:
+            mission=11; //Mission 10 Chateau, but basically mission 11
```

Comment:
I feel sad that you won't get chateau goals here lol, we can manually include them?